blockindex/contractstaking/event_handler.go

```diff
@@ -372,6 +373,7 @@ func newContractStakingEventHandler(cache *contractStakingCache, height uint64)
 	return &contractStakingEventHandler{
```

Comment:
L372 can be removed, `putHeight()` is called inside `func (s *Indexer) commit(handler *contractStakingEventHandler)` L179

Reply:
another small thing:
`func (s *contractStakingCache) Height()` should not return `s.height`, that is the height to be committed, it should read from DB and return the current height

Reply:
fixed